apps/src/templates/rubrics/AiAssessmentFeedback.jsx

```diff
@@ -0,0 +1,207 @@
+import React, {useState, useRef} from 'react';
+import PropTypes from 'prop-types';
+import i18n from '@cdo/locale';
+import style from './rubrics.module.scss';
+import {
+  BodyThreeText,
+  EmText,
+  Heading6,
+  StrongText,
+} from '@cdo/apps/componentLibrary/typography';
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+import Checkbox from '@cdo/apps/componentLibrary/checkbox/Checkbox';
+import Button from '@cdo/apps/templates/Button';
+
+export default function AiAssessmentFeedback({learningGoalKey}) {
+  const radioGroupName = `ai-assessment-feedback-${learningGoalKey}`;
+  const thumbsupval = 'thumbsup';
+  const thumbsdownval = 'thumbsdown';
+
+  const [aiFeedback, setAIFeedback] = useState('');
+  const [aiSubmitted, setAISubmitted] = useState(false);
+  const [aiFalsePos, setAIFalsePos] = useState(false);
+  const [aiFalseNeg, setAIFalseNeg] = useState(false);
+  const [aiVague, setAIVague] = useState(false);
+  const [aiFeedbackOther, setAIFeedbackOther] = useState(false);
+  const [aiOtherContent, setAIOtherContent] = useState('');
+  const [aiFeedbackReceived, setAIFeedbackReceived] = useState(false);
+
+  // Timer vars for feedback received flag
+  const receivedTimer = useRef();
+  const disappearAfter = 20000;
+
+  const radioAiFeedbackCallback = radioButtonData => {
+    setAIFeedback(radioButtonData);
+  };
+
+  const submitAiFeedbackCallback = () => {
+    const bodyData = JSON.stringify({
+      learningGoalKey: learningGoalKey,
+      aiFalseNeg: aiFalseNeg,
+      aiFalsePos: aiFalsePos,
+      aiVague: aiVague,
+      aiOther: aiFeedbackOther,
+      aiOtherContent: aiOtherContent,
+    });
+    // TODO: http put ai feedback json data
+    console.log(bodyData);
```

Comment:
I think this can be removed.

Reply:
Linting won't let me commit if `bodyData` isn't referenced somewhere 😢 (it will be when this is replaced with a http put)

Reply:
Weird and annoying!  That makes sense and as long as there is a ticket for that work, I think it is fine for now.